app/Http/Resources/MonsterListResource.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class MonsterListResource extends JsonResource
```

Comment:
Odd name, Resources are supposed to be singular, and have the `::collection()` method to basically convert it to a collection of resources. Is `MonsterResource` ok?

Reply:
There is a MonsterResource that gets all the information about a Monster and MonsterListResource that gets limited information for the purpose of Lists

Reply:
I don't want the list function to use the same resource as the retrieve function, because you could be getting data for hundreds of records and that makes for a heavy response @DanEvans485 what are your thoughts on this?